Jenkinsfile

```diff
@@ -40,4 +40,4 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipelineDAOS(['distros': ['centos7', 'centos8', 'leap15']])
\ No newline at end of file
+packageBuildingPipelineDAOS(['distros': ['centos7', 'centos8', 'leap15']])
```

Comment:
With all of the debian/ files being added, why is it not being added to this list to build?

Currently, DAOS is building on Ubuntu.  We don't want this to regress.